src/main/java/org/catmaid/ScaleCATMAID.java

```diff
@@ -149,113 +158,118 @@ final public static void scale(
 			final String tileFormat,
 			final int tileWidth,
 			final int tileHeight,
+			final long minX,
+			final long maxX,
+			final long minY,
+			final long maxY,
 			final long minZ,
 			final long maxZ,
 			final String format,
 			final float quality,
-			final int type ) throws Exception
+			final int type,
+			final boolean ignoreEmptyTiles) throws Exception
 	{
 		final BufferedImage alternative = new BufferedImage( tileWidth, tileHeight, BufferedImage.TYPE_INT_RGB );
-		
+
 		final int[] targetPixels = new int[ tileWidth * tileHeight ];
 		final BufferedImage target = new BufferedImage( tileWidth, tileHeight, BufferedImage.TYPE_INT_RGB );
 		
 		final BufferedImage sourceImage = new BufferedImage( tileWidth * 2, tileHeight * 2, BufferedImage.TYPE_INT_RGB );
 		final Graphics2D g = sourceImage.createGraphics();
 		final int[] sourcePixels = new int[ tileWidth * tileHeight * 4 ];
-				
-Z:		for ( long z = minZ; z <= maxZ; ++z )
+
+		for ( long z = minZ; z <= maxZ; z++ )
```

Comment:
++z does not store z before increment in a register and is therefore the right thing to do here